### ISSUE TITLE
Bug: upload deb SignatureDoesNotMatch

### ISSUE THREAD

**Issue by lukepolo** [oclif/oclif#1074]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
Bug

What is the current behavior?
-----------------------------
If the current behavior is a bug, please provide the steps to reproduce.

Env Varaibels Set

```
      AWS_S3_ENDPOINT: ${{ secrets.AWS_S3_ENDPOINT }}
      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
      AWS_S3_FORCE_PATH_STYLE: true
```

1. `oclif pack deb`
 
2. `oclif upload deb`

```
yarn run v1.22.19
$ oclif upload deb
oclif: starting upload of deb artifacts for v1.0.2-6268c76
oclif: s3:uploadFile ./dist/deb/codepier_1.0.2.6268c76-1_amd64.deb s3://cli/versions/1.0.2/6268c76/apt/codepier_1.0.2.6268c76-1_amd64.deb
oclif: s3:uploadFile ./dist/deb/codepier_1.0.2.6268c76-1_amd64.deb s3://cli/versions/1.0.2/6268c76/apt/./codepier_1.0.2.6268c76-1_amd64.deb
oclif: s3:uploadFile ./dist/deb/Packages.gz s3://cli/versions/1.0.2/6268c76/apt/Packages.gz
oclif: s3:uploadFile ./dist/deb/Packages.xz s3://cli/versions/1.0.2/6268c76/apt/Packages.xz
oclif: s3:uploadFile ./dist/deb/Packages.bz2 s3://cli/versions/1.0.2/6268c76/apt/Packages.bz2
oclif: s3:uploadFile ./dist/deb/Release s3://cli/versions/1.0.2/6268c76/apt/Release
oclif: s3:uploadFile ./dist/deb/Packages.gz s3://cli/versions/1.0.2/6268c76/apt/./Packages.gz
oclif: s3:uploadFile ./dist/deb/Packages.xz s3://cli/versions/1.0.2/6268c76/apt/./Packages.xz
oclif: s3:uploadFile ./dist/deb/Packages.bz2 s3://cli/versions/1.0.2/6268c76/apt/./Packages.bz2
oclif: s3:uploadFile ./dist/deb/Release s3://cli/versions/1.0.2/6268c76/apt/./Release
    SignatureDoesNotMatch: The request signature we calculated does not match the signature you provided. Check your key and signing method.
    Code: SignatureDoesNotMatch
error Command failed with exit code 1.
```

What is the expected behavior?
------------------------------
Please mention your oclif and OS version.

Ubuntu 20.04
oclif : 3.4.3

**Comment by lukepolo** [comment]
Found out the change in #1035 seems to have broke this

**Comment by lukepolo** [comment]
seems to be missing 386 build somehow


```
runner@linux-github-runner:~/codepier-cli$ yarn pack:deb
yarn run v1.22.19
$ oclif pack deb
oclif: gathering workspace for codepier to /home/runner/codepier-cli/tmp/codepier
oclif: will build 2 targets in parallel
oclif: building target codepier-v1.0.4-9c47c54-linux-x64.tar.gz
oclif: copying workspace ./tmp/codepier ./tmp/linux-x64/codepier
oclif: building target codepier-v1.0.4-9c47c54-linux-arm.tar.gz
oclif: copying workspace ./tmp/codepier ./tmp/linux-arm/codepier
oclif: downloading node-v18.13.0-linux-x64
oclif: downloading node-v18.13.0-linux-armv7l
oclif: extracting node-v18.13.0-linux-armv7l
oclif: extracting node-v18.13.0-linux-x64
building debian / x64
building debian / arm
dpkg --build "/home/runner/codepier-cli/tmp/apt/codepier_1.0.4.9c47c54-1_armel.apt" "/home/runner/codepier-cli/dist/deb/codepier_1.0.4.9c47c54-1_armel.deb"
                                                                                                                                                       dpkg --build "/home/runner/codepier-cli/tmp/apt/codepier_1.0.4.9c47c54-1_amd64.apt" "/home/runner/codepier-cli/dist/deb/codepier_1.0.4.9c47c54-1_amd64.deb"
                                                                                                                         finished building debian / arm
                                                                                                                                                       finished building debian / x64
debian packages created
                       debian packages archived
                                               debian packing complete
Done in 58.87s.

runner@linux-github-runner:~/codepier-cli$ yarn pack:deb:upload
yarn run v1.22.19
$ oclif upload deb
oclif: starting upload of deb artifacts for v1.0.4-9c47c54
oclif: s3:uploadFile ./dist/deb/codepier_1.0.4.9c47c54-1_amd64.deb s3://cli/versions/1.0.4/9c47c54/apt/codepier_1.0.4.9c47c54-1_amd64.deb
oclif: s3:uploadFile ./dist/deb/codepier_1.0.4.9c47c54-1_amd64.deb s3://cli/versions/1.0.4/9c47c54/apt/codepier_1.0.4.9c47c54-1_amd64.deb
oclif: s3:uploadFile ./dist/deb/Packages.gz s3://cli/versions/1.0.4/9c47c54/apt/Packages.gz
oclif: s3:uploadFile ./dist/deb/Packages.xz s3://cli/versions/1.0.4/9c47c54/apt/Packages.xz
oclif: s3:uploadFile ./dist/deb/Packages.bz2 s3://cli/versions/1.0.4/9c47c54/apt/Packages.bz2
oclif: s3:uploadFile ./dist/deb/Release s3://cli/versions/1.0.4/9c47c54/apt/Release
oclif: s3:uploadFile ./dist/deb/Packages.gz s3://cli/versions/1.0.4/9c47c54/apt/Packages.gz
oclif: s3:uploadFile ./dist/deb/Packages.xz s3://cli/versions/1.0.4/9c47c54/apt/Packages.xz
oclif: s3:uploadFile ./dist/deb/Packages.bz2 s3://cli/versions/1.0.4/9c47c54/apt/Packages.bz2
oclif: s3:uploadFile ./dist/deb/Release s3://cli/versions/1.0.4/9c47c54/apt/Release
oclif: done uploading deb artifacts for v1.0.4-9c47c54
Done in 1.40s.


runner@linux-github-runner:~/codepier-cli$ yarn pack:deb:promote --version 1.0.4 --sha 9c47c54
yarn run v1.22.19

$ oclif promote --deb --version 1.0.4 --sha 9c47c54
Promoting buildmanifests & unversioned tarballs to stable
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-linux-x64-buildmanifest to s3://cli/channels/stable/codepier-linux-x64-buildmanifest
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-linux-x64.tar.gz to s3://cli/channels/stable/codepier-linux-x64.tar.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-linux-arm-buildmanifest to s3://cli/channels/stable/codepier-linux-arm-buildmanifest
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-linux-arm.tar.gz to s3://cli/channels/stable/codepier-linux-arm.tar.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-win32-x64-buildmanifest to s3://cli/channels/stable/codepier-win32-x64-buildmanifest
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-win32-x64.tar.gz to s3://cli/channels/stable/codepier-win32-x64.tar.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-win32-x86-buildmanifest to s3://cli/channels/stable/codepier-win32-x86-buildmanifest
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-win32-x86.tar.gz to s3://cli/channels/stable/codepier-win32-x86.tar.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-darwin-x64-buildmanifest to s3://cli/channels/stable/codepier-darwin-x64-buildmanifest
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-darwin-x64.tar.gz to s3://cli/channels/stable/codepier-darwin-x64.tar.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-darwin-arm64-buildmanifest to s3://cli/channels/stable/codepier-darwin-arm64-buildmanifest
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/codepier-v1.0.4-9c47c54-darwin-arm64.tar.gz to s3://cli/channels/stable/codepier-darwin-arm64.tar.gz
Promoting debian artifacts to stable
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/codepier_1.0.4.9c47c54-1_amd64.deb to s3://cli/channels/stable/apt/codepier_1.0.4.9c47c54-1_amd64.deb
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/codepier_1.0.4.9c47c54-1_amd64.deb to s3://cli/channels/stable/apt/codepier_1.0.4.9c47c54-1_amd64.deb
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/codepier_1.0.4.9c47c54-1_i386.deb to s3://cli/channels/stable/apt/codepier_1.0.4.9c47c54-1_i386.deb
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/codepier_1.0.4.9c47c54-1_i386.deb to s3://cli/channels/stable/apt/codepier_1.0.4.9c47c54-1_i386.deb
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Packages.gz to s3://cli/channels/stable/apt/Packages.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Packages.gz to s3://cli/channels/stable/apt/Packages.gz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Packages.xz to s3://cli/channels/stable/apt/Packages.xz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Packages.xz to s3://cli/channels/stable/apt/Packages.xz
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Packages.bz2 to s3://cli/channels/stable/apt/Packages.bz2
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Packages.bz2 to s3://cli/channels/stable/apt/Packages.bz2
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Release to s3://cli/channels/stable/apt/Release
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Release to s3://cli/channels/stable/apt/Release
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/InRelease to s3://cli/channels/stable/apt/InRelease
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/InRelease to s3://cli/channels/stable/apt/InRelease
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Release.gpg to s3://cli/channels/stable/apt/Release.gpg
oclif: s3:copyObject from s3://cli/versions/1.0.4/9c47c54/apt/Release.gpg to s3://cli/channels/stable/apt/Release.gpg
    NoSuchKey: The specified key does not exist.
    Code: NoSuchKey
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


runner@linux-github-runner:~/codepier-cli/dist/deb$ ls -la
total 53380
drwxrwxr-x 2 runner runner     4096 Jan  8 18:44 .
drwxrwxr-x 7 runner runner     4096 Jan  8 18:43 ..
-rw-r--r-- 1 runner runner 28549644 Jan  8 18:44 codepier_1.0.4.9c47c54-1_amd64.deb
-rw-r--r-- 1 runner runner 26076280 Jan  8 18:44 codepier_1.0.4.9c47c54-1_armel.deb
-rw-rw-r-- 1 runner runner     1028 Jan  8 18:44 Packages
-rw-rw-r-- 1 runner runner      617 Jan  8 18:44 Packages.bz2
-rw-rw-r-- 1 runner runner      570 Jan  8 18:44 Packages.gz
-rw-rw-r-- 1 runner runner      624 Jan  8 18:44 Packages.xz
-rw-rw-r-- 1 runner runner     2025 Jan  8 18:44 Release
````

**Comment by cristiand391** [comment]
Hi @lukepolo 
Sorry for the late reply, we just pushed a release that will ignore the i386 build unless necessary (see #1140 )

For context, see:
https://github.com/oclif/oclif/issues/770#issuecomment-1508719530
**TL;DR**:
node dropped support for 32bit on linux after v9, unless you are hardcoding node v9 in your CLI you can't get a i386 deb installer.

**Comment by lukepolo** [comment]
Just tried new version (` "oclif": "^3.8.1",`) , seems to be still failing for me 


![image](https://user-images.githubusercontent.com/2066668/232559411-59d776b1-1a86-4592-984e-4fb64e7809d7.png)

**Comment by lukepolo** [comment]
i made a patch in my files and at least i can make it work ~ https://github.com/oclif/oclif/pull/1035 , im using miino so thats probably why its borked
